docs/ruleset_functions/expected_system_type_from_Table_G3_1_1a.md

```diff
@@ -0,0 +1,116 @@
+
+
+- check if the building area type is heated-only storage: `if building_area_type == "HEATED-ONLY_STORAGE":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 10: `expected_system_type = "SYS-10"`
+	- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 9: `expected_system_type = "SYS-9"
+
+
+- check if the building area type is Retail: `if building_area_type == "RETAIL":`
+	- check if there are fewer than 3 floors in the building: `if number_of_floors < 3:`
+		- set number_of_floors_string to " 1 or 2 floors": `number_of_floors_string = " 1 or 2 floors"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 4: `expected_system_type = "SYS-4"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 3: `expected_system_type = "SYS-3"
+	- otherwise reclasify the building area type to OTHER_NON_RESIDENTIAL: `else: building_area_type = "OTHER_NON_RESIDENTIAL"`
+
+
+- check if the building area type is Hospital: `if building_area_type == "HOSPITAL":`
+	- Hospital BAT doesn't use the climate zone to determine system type, so set climate_zone_category to "": `climate_zone_category = ""`
+	- check if the space area is more than 150,000 ft2 OR the building is more than 5 floors: `if((building_area > 150000) || (number_of_floors > 5)):`
+		- set building_area_string to " > 150,000 ft2 or > 5 floors": `building_area_string = " > 150,000 ft2 or > 5 floors"`
+		- the expected system is System 7: `expected_system_type = "SYS-7"`
+	- otherwise: `else:`
+		- set building_area_string to " All Other": `building_area_string = " All Other"`
+		- the expected system is System 5: `expected_system_type = "SYS-5"`
+
+
+- check if the building area type is other non-residential: `if building_area_type == "OTHER_NON_RESIDENTIAL":`
+	- check if the space area is less than 25,000: `if building_area < 25000:`
+		- set building_area_string to "< 25,000": `building_area_string = " < 25,000 ft2"`
+		- check if the building is fewer than 4 floors: `if number_of_floors < 4:
+			- set number_of_floors_string to " 3 floors or fewer": `number_of_floors_string = " 3 floors or fewer"`
+			- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+				- the expected system is System 4: `expected_system_type = "SYS-4"`
+			- otherwise, it's CZ_3b_3c_or_4_to_8: `if else:`
+				- the expected system is System 3: `expected_system_type = "SYS-3"
+		- else this is still < 25000 ft2, check if it is in a building less than 6 floors: `if number_of_floors < 6:`
+			- set number_of_floors_string to " 4-5 floors": `number_of_floors_string = " 4-5 floors"`
+			- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+				- the expected system is System 6: `expected_system_type = "SYS-6"`
+			- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+				- the expected system is System 5: `expected_system_type = "SYS-5"
```

Comment:
Need to add info on number of stories and "other nonresidential" to the "building_area_string"